src/database/models/ReviewComment.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Column,
+  ForeignKey,
+  Model,
+  Table,
+  DataType,
+  CreatedAt,
+  UpdatedAt,
+  BelongsTo,
+} from "sequelize-typescript"
+
+import { User } from "@database/models/User"
+
+import { ReviewRequest } from "./ReviewRequest"
+
+@Table({ tableName: "review_comments" })
+export class ReviewComment extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @ForeignKey(() => User)
+  @Column
+  reviewerId!: number
+
+  @ForeignKey(() => ReviewRequest)
+  @Column
+  reviewId!: string
+
+  @Column
+  comment!: string
+
+  @BelongsTo(() => User, {
+    onUpdate: "CASCADE",
+    onDelete: "CASCADE",
```

Comment:
are we doing soft deletes? We shouldnt be deleting the comments when a user is deleted, since we dont have that audit trail anymore

Reply:
@kishore03109  good point, will remove the cascade to keep the trail. But even so, for audit trail that means a User should never be deleted right?

By that logic, then cascading delete will make sense, because even if we retain this when a user is deleted, the relation to the table is gone. So how are we going to identify the user?

Reply:
I think regardless we shouldn't be hard deleting! for users there's a bunch of related information so we'll need to do a soft delete because of audit purposes

Reply:
@alexanderleegs Yes that's exactly my point - we shouldn't hard delete for User. Hence, this cascade delete of comments will never happen.

But to be sure, I will remove the cascade delete